integration_tests/steps/area_events.py

```diff
@@ -179,39 +178,34 @@ def both_sunny_profile(context):
             continue
         assert len(h2_trades) == 1
         assert h1_trades[0].offer.energy == h2_trades[0].offer.energy
-        profile_energy = sunny[time_slot] / (duration(hours=1) / slot_length)
-        assert h1_trades[0].offer.energy == profile_energy
+        assert h1_trades[0].offer.energy == sunny[time_slot]
 
 
 @then('House 1 PV follows the partially cloudy profile after 12:00')
 def house1_partially_cloudy(context):
-    slot_length = context.simulation.area.config.slot_length
     partial_cloud = _read_solar_profile(d3a_path + '/resources/Solar_Curve_W_partial.csv')
     house1 = next(filter(lambda x: x.name == "House 1", context.simulation.area.children))
 
     for time_slot, market in house1._markets.past_markets.items():
-        if time_slot.hour > 12:
+        if time_slot.hour < 12:
```

Comment:
Now the step description is not match any more, right?

Reply:
This test was not running up until now, and we had not figured out that the step definition was incorrect :(

Reply:
Ah right! NOW it is correct. I confused it. Thanks!